modules/ROOT/pages/lb-cert-upload.adoc

```diff
@@ -18,15 +21,41 @@ This file must contain the entire certificate chain ordered sequentially:
 -----END CERTIFICATE-----
 |===
 
-You don't need to include the root certificate in the certificate chain. However, ensure to include the ASCII armor in each certificate.
+You don't need to include the root certificate in the certificate chain.
+However, ensure that you include the ASCII armor in each certificate.
+
+
+== Add a Certificate to an Existing Load Balancer
+
+You can add a certificate to an existing DLB using Runtime Manager or the command-line interface.
+
+To add a certificate using Runtime Manager:
+
+. From Anypoint Platform, click *Runtime Manager*.
+. Click *Load Balancers*, then click the load balancer name.
+. In the *Certificates* tab, click *Add certificate*.
++
+.The arrow shows the *Add certificate* option in the *Certificates* tab.
+image::lb-cert-upload-update.png[Add certificate option in the load balancer settings page]
+. Select *Choose File* to upload the public key and private key files.
++
+For example:
+
```

Comment:
I suggest GLOBALLY replacing this phrase as an intro to an example, either by ending the previous sentence with a colon or with the phrase ", as shown here:" or "as shown in the following example:".